### ISSUE TITLE
appium/ruby_lib 7.0.0 is not compatible with Selenium WebDriver 2.48.0.

### ISSUE THREAD

**Issue by kirillzh** [appium/ruby_lib#353]
Getting `uninitialized constant Selenium::WebDriver::Remote::COMMANDS (NameError)` on driver creation.
Temporary solution is to lock `selenium-webdriver` version:
```ruby
# Gemfile
gem 'selenium-webdriver', '2.47.1'
```
```ruby 
# .gemspec
Gem::Specification.new do |spec|
  # ...
  spec.add_dependency 'selenium-webdriver', '2.47.1'
  #...
end
```

**Comment by titusfortner** [comment]
Oh, this is something I broke. There is now both Selenium::WebDriver::Remote::Bridge and Selenium::WebDriver::Remote::W3CBridge, so COMMANDS got put into the respective classes. I'll look at why this project requires that directly, but since Appium isn't using the W3C commands, it probably just needs to change to Selenium::WebDriver::Remote::Bridge::Commands

**Comment by bootstraponline** [comment]
The appium [debug method](https://github.com/appium/ruby_lib/blob/252e83806318df7df28e9060d3f8e1e56dc732ba/lib/appium_lib/common/patch.rb#L69) is brittle. It should probably be rewritten to alias the old method instead of using the code directly.

If people want to help out, I'm happy to [add more committers](https://github.com/appium/ruby_lib/issues/337).

**Comment by titusfortner** [comment]
Yeah, there's a lot of Monkey Patching going on. :)
Are there specs for this code? I can do a PR with the likely fix, but I don't run Appium tests to know if it for sure fixes the issue.

**Comment by titusfortner** [comment]
@kirillzh - can you verify this fixes the issue for 2.48?

**Comment by bootstraponline** [comment]
> Are there specs for this code?

There are specs for [android](https://github.com/appium/ruby_lib/tree/master/android_tests) and [ios](https://github.com/appium/ruby_lib/tree/master/ios_tests) but not for this.

**Comment by kirillzh** [comment]
Well, it does work with 2.48, but breaks previous versions (if you lock 'selenium-webdriver').

**Comment by bootstraponline** [comment]
[this](https://github.com/appium/ruby_lib/commit/c89464f22314d7521292f2b3e9367e53f1c5f00e) will ensure the gem has the latest selenium-webdriver.

**Comment by kirillzh** [comment]
@bootstraponline, do we need to bump version? Gem is not updated on RubyGems yet.
Thanks for the fix!

**Comment by bootstraponline** [comment]
I released appium_lib (8.0.0) to rubygems with the fix.

**Comment by kirillzh** [comment]
@bootstraponline, awesome!
